migration/migrator/migrations/master/20230314133718_user_pronouns.py

```diff
@@ -0,0 +1,108 @@
+"""Migration for the Submitty master database."""
```

Comment:
Put the 2 master migrations together into 1 file. It's more clear to developers if each PR makes 1 migration file per category.

Reply:
Done